plugins/kosync.koplugin/main.lua

```diff
@@ -250,7 +282,22 @@ function KOSync:syncToProgress(progress)
     end
 end
 
-function KOSync:updateProgress()
+local function promptLogin()
+    UIManager:show(InfoMessage:new{
+        text = _("Please register / login before using progress synchronization feature."),
+        timeout = 3,
+    })
+end
+
+local function showSyncError()
+    UIManager:show(InfoMessage:new{
+        text = _("Something went wrong when syncing progress, " ..
+                 "please check your network connection and try later."),
```

Comment:
Our gettext module cannot extract the final string to be translated.
It will get:
```
"Something went wrong when syncing progress, \" ..\n"
"                 \"please check your network connection and try later."
```
which will confuse the translators. 

Better to combine the string in the code.